packages/dd-trace/src/metrics.js

```diff
@@ -259,13 +261,29 @@ function captureHistograms () {
   })
 }
 
+/**
+ * Gathers and reports Event Loop Utilization (ELU) since last run
+ *
+ * ELU is a measure of how busy the event loop is, like running JavaScript or
+ * waiting on *Sync functions. The value is between 0 (idle) and 1 (exhausted).
+ *
+ * performance.eventLoopUtilization available in Node.js >= v14.10, >= v12.19, >= v16
+ */
+const captureELU = ('eventLoopUtilization' in performance) ? () => {
```

Comment:
This might be an over-optimization but originally the code checks for support with each run. Now it sets a noop on startup if the Node.js version doesn't support the API.